liberty-maven-plugin/src/main/java/io/openliberty/tools/maven/server/DevMojo.java

```diff
@@ -108,9 +112,14 @@
     private int runId = 0;
 
     private ServerTask serverTask = null;
+    
+    private boolean usingBoost = false;
 
     @Component
     private BuildPluginManager pluginManager;
+    
+    @Component
+	protected ProjectBuilder mavenProjectBuilder;
```

Comment:
Throughout the file, use 4 spaces instead of tabs.

Reply:
Fixed.